index.bs

```diff
@@ -384,6 +385,16 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         of frames used in one second (frames per second). It is represented
         either as a double or as a fraction.
       </p>
+
+      <p>
+        The <dfn for='VideoConfiguration' dict-member>hasAlphaChannel</dfn> member
+        represents whether the video track contains alpha channel information. If
+        true, the encoded video stream can produce per-pixel alpha channel information
+        when decoded. If false, the video stream cannot produce per-pixel alpha channel
+        information when decoded. If undefined, the UA should determine whether the
+        video stream encodes alpha channel information based on the indicated
+        {{VideoConfiguration/contentType}}, if possible.
```

Comment:
I like this, but would append a final sentence: Otherwise, the value should default to false.